txnkv/transaction/commit.go

```diff
@@ -212,13 +220,18 @@ func (action actionCommit) handleSingleBatch(c *twoPhaseCommitter, bo *retry.Bac
 					zap.Error(err),
 					zap.Uint64("txnStartTS", c.startTS),
 					zap.Uint64("commitTS", c.commitTS),
-					zap.Strings("keys", hexBatchKeys(keys)))
+					zap.Strings("keys", hexBatchKeys(keys)),
+					zap.Uint64("sessionID", c.sessionID),
+					zap.String("debugInfo", tikverr.ExtractDebugInfoStrFromKeyErr(keyErr)))
```

Comment:
Also need to print the information for async commit path if secondary key commit fails.